Assets/Scripts/Game/EnemyMotor.cs

```diff
@@ -1385,11 +1409,14 @@ void ApplyFallDamage()
                     }
                 }
 
-                lastGroundedY = transform.position.y;
+                LastGroundedY = transform.position.y;
             }
             // For flying enemies, "lastGroundedY" is really "lastAltitudeControlY"
             else if (flies && !flyerFalls)
```

Comment:
Now I feel I gotta nitpick and ask you to put write this as `if((flies && !flyerFalls) || IsLevitating || entity.IsSlowFalling)`

Reply:
Done.